packages/jaeger-ui/src/model/ddg/DdgVertex.test.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import DdgVertex from './DdgVertex';
+import { DdgEdge } from './types';
+
+describe('DdgVertex', () => {
+  function makeTestVertex(vertexKey) {
+    const vertex = new DdgVertex({ key: vertexKey });
+    vertex.pathElems.add(`${vertexKey}-pathElem0`);
+    vertex.pathElems.add(`${vertexKey}-pathElem1`);
+    return vertex;
+  }
+
+  function makeTestEdge(from, to) {
+    const edge = new DdgEdge({ from, to });
+    from.egressEdges.set(to, edge);
+    to.ingressEdges.set(from, edge);
+  }
+
+  const twoHopUpstreamVertexKey = 'twoHopUpstreamVertexKey';
```

Comment:
these keys don't need to be `const`s they can just be the argument to `makeTestVertex` directly...